pkgs/top-level/all-packages.nix

```diff
@@ -16499,6 +16499,8 @@ in
 
   mono-addins = callPackage ../development/libraries/mono-addins { };
 
+  movine = callPackage ../development/tools/database/movine { };
```

Comment:
```suggestion
  movine = callPackage ../development/tools/database/movine {
    postgresql = postgresql_12;
    inherit (darwin.apple_sdk.frameworks) Security;
  };
```

Reply:
I was not able to build this with unstable nix any more, so I discarded those changes.
Is this strictly necessary? I will have to look into why it fails on unstable, in that case.

Reply:
Checking out your PR & applying the suggested changes evaluates fine for me.

![grafik](https://user-images.githubusercontent.com/23431373/116230343-d3108280-a757-11eb-8659-71eac389657f.png)

If you want to `callPackage` the default.nix yourself, then you also need to replicate what the `callPackage` in all-packages.nix does.

`nix-build -E 'with import <nixpkgs> {}; callPackage ./default.nix { postgresql = postgresql_12; inherit (darwin.apple_sdk.frameworks) Security; }'`

Reply:
ah, yeah, good point lmao
works fine now, cheers